src/components/Tabs/stories/Tabs.intro.stories.mdx

```diff
@@ -0,0 +1,35 @@
+import { ArgsTable, Canvas, Meta } from '@storybook/addon-docs';
+import TabsRB from 'react-bootstrap/Tabs';
+import Accordion from 'react-bootstrap/Accordion';
+
+import { Tabs } from '../..';
+
+<Meta title="Components/Tabs/Intro" component={Tabs} />
+
+# Tabs
+
+Display images and other content in a dialog box, either individually or as part of a gallery.
+Implements the<a href="https://www.w3.org/TR/wai-aria-practices/#dialog_modal" target="_blank"> WAI-ARIA Dialog (Modal) design pattern</a>
+
+When the screen width is less than 992px, the tabs will reformat into an accordion style to better fit. <br />Try it now by opening `inspect element` and increasing its width!
+
+## Example
+
+<Canvas>
+  <Tabs
+    id="uniquevalue"
+    panels={[
+      { title: 'tab one', content: <p>Tab Content</p>, id: 'text1' },
```

Comment:
It would be nice if you could follow the capitalization and format of the WET style guide.

![image](https://user-images.githubusercontent.com/31704878/173981479-2b0ec6d0-45fe-4ba7-b339-fffa622580eb.png)